hub-js/graphql/local/schema.graphql

```diff
@@ -34,16 +39,27 @@ scalar LockOwnerID
 
 type TypeInstance {
   id: ID! @id
-
+  createdAt: DateTime
   lockedBy: LockOwnerID
 
   """
   Common properties for all TypeInstances which cannot be changed
   """
   typeRef: TypeInstanceTypeReference!
     @relation(name: "OF_TYPE", direction: "OUT")
-  uses: [TypeInstance!]! @relation(name: "USES", direction: "OUT")
-  usedBy: [TypeInstance!]! @relation(name: "USES", direction: "IN")
+  """
+  Return TypeInstance that are used. List is sorted by TypeInstance's TypeRef path in ascending order.
```

Comment:
```suggestion
  Return TypeInstances that are used. List is sorted by TypeInstance's TypeRef path in ascending order.
```

should I duplicate that information to `uses` field and `typeInstances` query?

Reply:
```suggestion
  Returns TypeInstances that are used. List is sorted by TypeInstance's TypeRef path in ascending order, and then by revision in descending order.
```

Yes, please duplicate that, as then it will be visible in the documentation for both fields. Thanks!